examples/accounts/scripts/02-accounts.yaml

```diff
@@ -3,7 +3,6 @@
 # $ pact -a examples/accounts/mock-system.yaml | curl -d @- http://localhost:8080/api/v1/send
 # $ pact -a examples/accounts/accounts.yaml | curl -d @- http://localhost:8080/api/v1/send
 
-
```

Comment:
The accounts regression shouldn't be modified for this, go ahead and copy all these scripts into a `resume` folder in `tests`